WalletWasabi.Tests/IntegrationTests/LiveServerTests.cs

```diff
@@ -17,15 +17,23 @@
 namespace WalletWasabi.Tests.IntegrationTests
 {
 	[Collection("LiveServerTests collection")]
-	public class LiveServerTests
+	public class LiveServerTests : IAsyncLifetime
 	{
 		public LiveServerTests(LiveServerTestsFixture liveServerTestsFixture)
 		{
 			LiveServerTestsFixture = liveServerTestsFixture;
+		}
 
+		public async Task InitializeAsync()
+		{
 			var torManager = new TorProcessManager(Global.Instance.TorSocks5Endpoint, Global.Instance.TorLogsFile);
 			torManager.Start(ensureRunning: true, dataDir: Path.GetFullPath(AppContext.BaseDirectory));
-			Task.Delay(3000).GetAwaiter().GetResult();
+			await Task.Delay(3000);
```

Comment:
Can we ask TorProcessManager is Tor is running or not?

Reply:
Not sure if it is helpful at this point because my PRs attempt to reach the state where we would just call: `await torManager.Start(ensureRunning: true)` and that would wait for Tor to be running or throw an exception that Tor is not reachable.